services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/IslReplugSpec.groovy

```diff
@@ -24,6 +26,7 @@ class IslReplugSpec extends BaseSpecification {
     @Value('${opentsdb.metric.prefix}')
     String metricPrefix
 
+    @Tags(SMOKE)
```

Comment:
Not interesting. Replugs on prod happen __very__ rarely. More interesting is the `New potential self-loop ISL (different ports on the same switch) is not getting discovered when replugging` in this spec, because it
1. Verifies crucial requirement to not discover self-looped ISLs
2. Performs additional statistic verifications, verifies that stats for default rules are getting logged in otsdb

Reply:
this test was removed form set of smoke test

and mentioned test was added